src/containers/Plot/index.tsx

```diff
@@ -26,11 +28,11 @@ class Plot extends React.Component<PlotProps, {}> {
     }
 
     public render(): JSX.Element | null {
-        const { plotConfig, time } = this.props;
+        const { plotConfig, time, firstFrameTime } = this.props;
         const { shouldRenderTimeIndicator, data, layout } = plotConfig;
         const TIME_INDICATOR_LINE = "timeIndicatorLine";
 
-        if (shouldRenderTimeIndicator && time !== 0) {
+        if (shouldRenderTimeIndicator && time !== firstFrameTime) {
```

Comment:
So this is keeping the time plot from being rendered when the plots are first rendered and time is at start time, but if you play through, and then get to time 0/startTime again, won't the time plot still be rendered? Is this bad? if it is, the time plots should also be removed from the data when time is reset to start time

Reply:
Oh you're totally right..... That would create a bug. This check `time !== 0` was only for a very subtle cosmetic thing, where it made the y-axis look ever so slightly thicker at t = 0 when the plots first loaded. I think I'm just going to take it out altogether for this PR and if it bugs us we can fix it later? What do you think?

Reply:
seems fine to me